client/my-sites/site-settings/navigation.jsx

```diff
@@ -29,6 +29,7 @@ export class SiteSettingsNavigation extends Component {
 			discussion: translate( 'Discussion', { context: 'settings screen' } ),
 			traffic: translate( 'Traffic', { context: 'settings screen' } ),
 			security: translate( 'Security', { context: 'settings screen' } ),
+			backups: translate( 'Backups', { context: 'backups screen' } ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Backups' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/rewind-credentials)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.